app/services/save_keywords_to_db.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class SaveKeywordsToDb
```

Comment:
@aetheryna We can dismiss this if we let the model save our keywords list 🙌
Discussion from [here](https://github.com/aetheryna/google-scraper-ruby/pull/34/files#r997908113)

Reply:
Done 😄 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/34/commits/439908c2734f5d8ffecd8a64310ecddee29f9cf6